extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/handler/ChaserThingHandler.java

```diff
@@ -0,0 +1,207 @@
+
+    private List<Channel> channels = new ArrayList<Channel>();
+    private List<ValueSet> values = new ArrayList<ValueSet>();
+
+    private boolean resumeAfter = false;
+    private OnOffType isRunning = OnOffType.OFF;
+
+    public ChaserThingHandler(Thing dimmerThing) {
+        super(dimmerThing);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        switch (channelUID.getId()) {
+            case CHANNEL_SWITCH:
+                if (command instanceof OnOffType) {
+                    if (((OnOffType) command).equals(OnOffType.ON)) {
+                        Integer channelCounter = 0;
+                        for (Channel channel : channels) {
+                            if (resumeAfter) {
+                                channel.suspendAction();
+                            } else {
+                                channel.clearAction();
+                            }
+                            for (ValueSet value : values) {
+                                channel.addChannelAction(new FadeAction(value.getFadeTime(),
+                                        value.getValue(channelCounter), value.getHoldTime()));
+                            }
+                            if (resumeAfter) {
+                                channel.addChannelAction(new ResumeAction());
+                            }
+                            if (isLinked(channelUID.getId())) {
+                                channel.addListener(channelUID, this, ListenerType.ACTION);
+                            }
+                            channelCounter++;
+
+                        }
+                    } else {
+                        for (Channel channel : channels) {
+                            if (resumeAfter && channel.isSuspended()) {
+                                channel.setChannelAction(new ResumeAction());
+                            } else {
+                                channel.clearAction();
+                            }
+                        }
+                    }
+                } else if (command instanceof RefreshType) {
+                    updateState(channelUID, isRunning);
+                } else {
+                    logger.warn("command {} not supported in channel {}:switch", command.getClass(),
+                            this.thing.getUID());
+                }
+                break;
+            case CHANNEL_CONTROL:
+                if (command instanceof StringType) {
+                    Vector<ValueSet> oldValues = new Vector<ValueSet>(values);
+                    if (parseChaserConfig(((StringType) command).toString())) {
+                        logger.debug("updated chase config in {}", this.thing.getUID());
+                    } else {
+                        // restore old chase config
+                        values = oldValues;
+                        logger.warn("could not update chase config in {}, malformed", this.thing.getUID());
```

Comment:
I understood that error should only be used for things that could be dangerous for ESH as whole. This is not the case here. If you think this should be changed anyway, I'll do so.